src/ovms.h

```diff
@@ -501,6 +501,16 @@ OVMS_Status* OVMS_ServableMetadataGetInput(OVMS_ServableMetadata* metadata, uint
 // \return OVMS_Status object in case of failure
 OVMS_Status* OVMS_ServableMetadataGetOutput(OVMS_ServableMetadata* metadata, uint32_t id, const char** name, OVMS_DataType* datatype, size_t* dimCount, int64_t** shapeMinArray, int64_t** shapeMaxArray);
 
+
+// #ifdef EXPERIMENTAL / MEDIAPIPE
```

Comment:
Why should this be removed without  mediapipe? RTInfo is not related to mediapipe and can exist inside OV model outside of mediapipe as well.